src/govuk/i18n.mjs

```diff
@@ -37,12 +37,23 @@ I18n.prototype.t = function (lookupKey, options) {
     // Get the plural suffix
     var pluralSuffix = this.getPluralSuffix(options.count)
 
-    // Overwrite our existing lookupKey
-    lookupKey = lookupKey + '.' + pluralSuffix
+    // Use the correct plural form if provided
+    if (lookupKey + '.' + pluralSuffix in this.translations) {
+      lookupKey = lookupKey + '.' + pluralSuffix
+    // Fall back to `other` if the plural form is missing, but log a warning
+    // to the console
+    } else if (lookupKey + '.other' in this.translations) {
+      if (console && 'warn' in console) {
+        console.warn('i18n: Missing plural form ".' + pluralSuffix + '" for "' +
```

Comment:
The message formatting seems a bit inconsistent as it refers to the missing key as ".many" but then says it's falling back to the \`other\` key. I'd expect the missing key to probably also be written as \`many\`. 

Otherwise this all looks good to me. Seems to do the job.

Reply:
Good catch! 👀